src/trace_processor/trace_processor_shell.cc

```diff
@@ -705,7 +705,7 @@ struct CommandLineOptions {
 
   std::string query_file_path;
   std::string query_string;
-  std::string sql_module_path;
+  std::vector<std::string> sql_module_paths;
```

Comment:
This and below need to be renamed as well.

Reply:
Now that we are here, I made more renames of module -> package in the file to reflect that
package is the root dir to host subdirs or SQL source files.
module is an individual SQL source file.

PTAL. Thanks.